samples/generated/express-web-no-oidc/README.md

```diff
@@ -0,0 +1,28 @@
+# express-web-no-oidc sample
+
+This sample demonstrates using `@okta/okta-auth-js` in NodeJS. The `signIn` method is used to authenticate users given a username and password. On success, the `signIn` method returns a transaction object containing a `sessionToken`. The `sessionToken` can be used to establish an Okta session.
```

Comment:
nit: add link for sessionToken (https://developer.okta.com/docs/reference/api/sessions/#session-token)